decidim-core/app/assets/stylesheets/decidim/_decidim.scss

```diff
@@ -1,6 +1,6 @@
 @import "variables";
-@import "utils/*";
 @import "foundation";
+@import "utils/*";
```

Comment:
foundation styles ought to be added before utils (mixins, to be exact) in order to get the `$foundation-palette` variable